swf/src/write.rs

```diff
@@ -2136,7 +2152,7 @@ impl<W: Write> Writer<W> {
             let mut has_wide_offsets = false;
             let has_wide_codes = !font.is_ansi;
             let mut shape_buf = Vec::new();
-            {
+            if font.version == 2 {
```

Comment:
I'm not sure about splitting this up between version; I can have a DefineFont2 tag without any glyphs or offset table (set to device font).